tests/workflows/integration/test-rock/.trivyignore

```diff
@@ -1,2 +1,3 @@
 # Pebble CVE, false positive: https://github.com/canonical/pebble/issues/498#issuecomment-2339325722
 CVE-2024-34156
+CVE-2024-45338
```

Comment:
Can you add a comment that this can be removed after https://github.com/canonical/pebble/pull/540 has been released?

Reply:
Added, thanks!